hypothesis-python/tests/cover/test_lookup.py

```diff
@@ -1004,3 +1024,19 @@ def f(x: int):
     msg = "@no_type_check decorator prevented Hypothesis from inferring a strategy"
     with pytest.raises(TypeError, match=msg):
         st.builds(f).example()
+
+
+@pytest.mark.skipif(
+    importlib.util.find_spec("numpy") is None,
+    reason="numpy is not available",
+)
+def test_numpy_type():
```

Comment:
I'd put this in the numpy tests, parametrize it over all the dtype types, and maybe a `# pragma: no cover` if the implementation needs it.

Reply:
Yes, that test is already there (more or less), the only point if this test would be to cover the numpy-not-installed case. That is trivial enough that it can be no-covered, agreed.